rules/scala.bzl

```diff
@@ -182,6 +206,7 @@ def _scala_library_implementation(ctx):
         ("javainfo", _phase_javainfo),
         ("compile", _phase_noop),
         ("singlejar", _phase_singlejar),
+        ("coverage", _phase_coverage_jacoco),
```

Comment:
As far as I can tell, `bazel coverage` only works against test targets, why would we need for `scala_library`? I apologize if I am wrong.

Reply:
The coverage phase is responsible for adding the jacoco instrumentation actions to the action graph.